plugins/module_utils/mso.py

```diff
@@ -207,6 +207,8 @@ def mso_service_graph_node_spec():
 def mso_service_graph_node_device_spec():
     return dict(
         name=dict(type="str", required=True),
+        provider_connector_type=dict(type="bool", default=False),
```

Comment:
What is the reason of enforcing this default, because we typically do not set defaults in argument spec?